openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -10,18 +10,30 @@ <h5 class="{{ class_prefix }}__display__title">
         </header>
     {% endif %}
 
-    <div class="{{ class_prefix }}__display__file {% if not file_url %}is--hidden{% endif %} submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
-        {% if file_upload_type == "image" %}
-            <img class="submission__answer__file submission--image"
-                 alt="{% trans "The image associated with this submission." %}"
-                 src="{{ file_url }}" />
-        {% elif file_upload_type == "pdf-and-image" or file_upload_type == "custom" %}
-            <a href="{{ file_url }}" class="submission__answer__file submission--file" target="_blank">
-                {% trans "View the file associated with this submission." %}
-            </a>
-            {% if show_warning %}
-                <p class="submission_file_warning">{% trans "(Caution: This file was uploaded by another course learner and has not been verified, screened, approved, reviewed, or endorsed by the site administrator. If you decide to access it, you do so at your own risk.)" %}</p>
+    <div class="{{ class_prefix }}__display__file {% if not file_urls %}is--hidden{% endif %} submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
+        <div class="submission__answer__files">
+        {% for file_url, file_description in file_urls %}
+            <div class="submission__answer__file__block submission__answer__file__block__{{ forloop.counter0 }}">
+            {% if file_upload_type == "image" %}
+                {% if file_description %}
+                <div class="submission__file__description__label" id="file_description_{{ xblock_id }}_{{ forloop.counter0 }}">{{ file_description }}:</div>
```

Comment:
@efischer19 Good find. Thanks. I'll add `including_template` into templates. `xblock_id` is still necessary because it will be useful in case more than one ORA block on the same page